aws_lambda_builders/workflows/ruby_bundler/validator.py

```diff
@@ -0,0 +1,62 @@
+"""
+Ruby Runtime Validation
+"""
+
+import logging
+import os
+import subprocess
+
+from aws_lambda_builders.exceptions import MisMatchRuntimeError
+
+LOG = logging.getLogger(__name__)
+
+
+class RubyRuntimeValidator(object):
```

Comment:
What's the purpose of this Validator? The ruby workflow uses bundler not ruby. Shouldn't we be validating the binaries the workflow uses?

Reply:
Yeah agreed on this, we need a best attempt bundler validator instead.